ee/codegen/python_file_merging/tests/fixtures/nodes/node_comments/generic_node.json

```diff
@@ -0,0 +1,10 @@
+{
+  "outputs": [
```

Comment:
In the node comments test, the snapshot fails due to a missing node comment and attribute, not due to the outputs missing. Iterate on this fixture to get this test to pass.

Once you notice how to pass this test, apply your learnings on the rest of the tests